src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1762,7 +1763,8 @@ public override void PopulateFromMLFields(List<object> fields)
 
         public override void Execute(ICpu cpu)
         {
-            cpu.PushStack(cpu.MakeUserDelegate(EntryPoint));
+            IUserDelegate pushMe = cpu.MakeUserDelegate(EntryPoint);
+            cpu.PushStack(pushMe);
```

Comment:
This got split into two lines because I was doing a debug print of the pushMe value.  It doesn't hurt to leave it like this, but I thought I'd mention why just so it wasn't confusing.